virttest/virt_admin.py

```diff
@@ -62,8 +62,9 @@
 try:
     VIRTADMIN_EXEC = path.find_command("virt-admin")
 except path.CmdNotFoundError:
-    logging.warning("virt-admin executable not set or found on path, "
-                    "virtadmin-admin module will not function normally")
+    logging.getLogger('avocado.app').warning(
```

Comment:
@PaulYuuu So I guess these warnings are not included in the job log? Or where does the `avocado.app` logger's handlers point to?

Reply:
`avocado.app` is same as `LOG_UI`
https://github.com/avocado-framework/avocado/blob/41b5626cf2d409110338486db2489d8d224f3bab/avocado/core/output.py#L34

You are right, they are not included in the job.log. Actually, before avocado is initialized, messages called by logging will not be saved to any log files if I understand correctly, even use `LOG_JOB`
https://github.com/avocado-framework/avocado/blob/41b5626cf2d409110338486db2489d8d224f3bab/avocado/core/output.py#L36

Reply:
> `avocado.app` is same as `LOG_UI`
> https://github.com/avocado-framework/avocado/blob/41b5626cf2d409110338486db2489d8d224f3bab/avocado/core/output.py#L34

Alright, then this is indeed better approach.

> You are right, they are not included in the job.log. Actually, before avocado is initialized, messages called by logging will not be saved to any log files if I understand correctly, even use `LOG_JOB`
> https://github.com/avocado-framework/avocado/blob/41b5626cf2d409110338486db2489d8d224f3bab/avocado/core/output.py#L36

My only remaining worry is if some of these modules affect people that don't use them. I can do some testing once we have more reviews, feel free to ping me then.